sigmastate/src/test/scala/special/sigma/SigmaDslSpecification.scala

```diff
@@ -5144,26 +5144,47 @@ class SigmaDslSpecification extends SigmaDslTesting
         FixedCostItem(SAvlTree.isUpdateAllowedMethod, FixedCost(JitCost(15)))
       )
     )
-    verifyCases(
-      Seq(ctx -> Expected(Success(ctx.LastBlockUtxoRootHash.isUpdateAllowed), cost = 1787, isUpdateAllowedCostDetails, 1787)),
-      existingFeature(
-        { (x: Context) => x.LastBlockUtxoRootHash.isUpdateAllowed },
-        "{ (x: Context) => x.LastBlockUtxoRootHash.isUpdateAllowed }",
-        FuncValue(
-          Vector((1, SContext)),
-          MethodCall.typed[Value[SBoolean.type]](
-            MethodCall.typed[Value[SAvlTree.type]](
-              ValUse(1, SContext),
-              SContext.getMethodByName("LastBlockUtxoRootHash"),
+
+    if (lowerMethodCallsInTests){
+      verifyCases(
+        Seq(ctx -> Expected(Success(ctx.LastBlockUtxoRootHash.isUpdateAllowed), cost = 1787, isUpdateAllowedCostDetails, 1787)),
+        existingFeature(
+          { (x: Context) => x.LastBlockUtxoRootHash.isUpdateAllowed },
+          "{ (x: Context) => x.LastBlockUtxoRootHash.isUpdateAllowed }",
+          FuncValue(
+            Vector((1, SContext)),
+            MethodCall.typed[Value[SBoolean.type]](
+              LastBlockUtxoRootHash,
+              SAvlTree.getMethodByName("isUpdateAllowed"),
               Vector(),
               Map()
-            ),
-            SAvlTree.getMethodByName("isUpdateAllowed"),
-            Vector(),
-            Map()
+            )
           )
-        )),
-      preGeneratedSamples = Some(samples))
+        ),
+        preGeneratedSamples = Some(samples)
+      )
+    } else {
+      verifyCases(
+        Seq(ctx -> Expected(Success(ctx.LastBlockUtxoRootHash.isUpdateAllowed), cost = 1787, isUpdateAllowedCostDetails, 1787)),
+        existingFeature(
+          { (x: Context) => x.LastBlockUtxoRootHash.isUpdateAllowed },
+          "{ (x: Context) => x.LastBlockUtxoRootHash.isUpdateAllowed }",
+          FuncValue(
+            Vector((1, SContext)),
+            MethodCall.typed[Value[SBoolean.type]](
+              MethodCall.typed[Value[SAvlTree.type]](
+                ValUse(1, SContext),
+                SContext.getMethodByName("LastBlockUtxoRootHash"),
+                Vector(),
+                Map()
+              ),
+              SAvlTree.getMethodByName("isUpdateAllowed"),
+              Vector(),
+              Map()
+            )
+          )),
+        preGeneratedSamples = Some(samples))
+    }
```

Comment:
Look how I did parameterization without unnecessary duplication. Only the expected tree depends on lowerMethodCallsInTests.